department-of-veterans-affairs/caseflow
# APPEALS-6071: Adding PCAFC Issues to Intake Add Issues Dropdown

Resolves [APPEALS-6071](https://vajira.max.gov/browse/APPEALS-6071)

### Description
Add more granular issue types for Caregiver issues underneath the VHA benefit type on the Add/Remove Issues page in Caseflow Intake.

### Acceptance Criteria
- [x] Remove current Caregiver issue label
- [x] Add the following issues types to the VHA issues dropdown:
   - Caregiver – Eligibility
   - Caregiver – Tier Level
   - Caregiver – Revocation/Discharge
   - Caregiver – Other(Identify the need, not the technical solution)
- [x] Add/Update any relevant tests

### Testing Plan
1. Log into Caseflow as an Intake user (ex: `BVAISHAW`)
2. Navigate to Caseflow Intake (`/intake`) and select "Decision Review Request: Board Appeal (Notice of Disagreement) — VA Form 10182".
3. Enter the following VA file number: `123456789`
4. Follow the 10182 input flow until you get to the Add/Remove Issues page.
5. Click the `Add Issue` button:
<img width="535" alt="add_issue_button" src="https://user-images.githubusercontent.com/99351305/177413069-f2a82b58-b04a-4014-9c39-1807b9b8b44d.PNG">
6. Select "Veteran Health Administration" from the "Benefit type" dropdown.
7. Note the presence of the updated Caregiver issue types (see the AC above), and the absence of the issue type entitled just "Caregiver".
8. Select one of the new Caregiver issue types. Enter any description for the issue you'd like.
9. Click the "Add this Issue" button.
10. Click the "Submit appeal" button.
11. Click on the "Correct these issues" link on the completion page.
12. Grab the UUID of the appeal you've just created from the URL path. The UUID will be located as follows: `/appeals/<This is the UUID>/edit`

ex: `appeals/ac6dbd74-9f70-480b-a67e-3fca05a61c49/edit`

13. Navigate to the case details page for the appeal (`/queue/appeals/<UUID>`)
14. Scroll down to the "Issues" section on the case details page. Confirm that the issue type is the one you've selected in step 8.
<img width="785" alt="request_issue_visible" src="https://user-images.githubusercontent.com/99351305/177414095-9f3debc4-09e0-441c-9abd-cd1d1367d5b2.PNG">

15. Feel free to repeat these testing steps for the 3 other new issue types, as well as some non-caregiver ones for regression testing.


### User Facing Changes

 BEFORE|AFTER
 ---|---
![localhost_3000_intake_add_issues (3)](https://user-images.githubusercontent.com/99351305/177409081-0c8256a0-4e6b-4319-ba7d-52ce5724c053.png)|![localhost_3000_intake_add_issues (2)](https://user-images.githubusercontent.com/99351305/177409102-4b6fc579-c1aa-4e9f-80b0-2b7ae9d2dcb8.png)
